.env

```diff
@@ -0,0 +1,50 @@
+# Set all your environment variables here
+
+
+#########################################
+## GENERAL
+#########################################
+
+HOSTNAME=localhost
+# Adapt to your production domain and use a reverse proxy
+#HOSTNAME=taiga.company.com
+
+PORT=80
+
+TAIGA_DB_PWD=somethingverysecure,changethis!
+
+
+#########################################
+## SECURITY
+#########################################
+
+TAIGA_SECRET=somethingevenmoresecure!!really,changethis!
+
+TAIGA_ADMIN_PASSWORD=betterthan123123
+
+#########################################
+## EMAIL
+#########################################
+
+TAIGA_EMAIL_DOMAIN=company.com
+TAIGA_SMTP_USER=
+TAIGA_SMTP_PWD=
```

Comment:
@AminovE99 Setup this to use a local [mailcatcher](https://github.com/sj26/mailcatcher) instance.